code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -77,28 +77,44 @@
 		return
 	. = ..()
 
+/obj/structure/chair/update_atom_colour()
+	. = ..()
+	if (armrest)
+		color_atom_overlay(armrest)
+
+/obj/structure/chair/color_atom_overlay(mutable_appearance/overlay)
+	overlay.color = color
+	if (!cached_color_filter)
+		return overlay
+	to_chat(world, "[overlay.icon_state]")
```

Comment:
hey uhhh is this supposed to still be in here

Reply:
:)